src/navigation/index.tsx

```diff
@@ -41,6 +42,10 @@ const screens: NavScreen[] = [
     name: SCREEN_NAMES.common.events.planningDetails,
     component: PlanningDetails,
   },
+  {
+    name: SCREEN_NAMES.common.events.ceContactDetails,
```

Comment:
what's "ceContactDetails"? Is this a typo?

Reply:
ce stands for "create-edit" because the screen will be used in both conditions